### ISSUE TITLE
Two finger scrolling on Firefox not as responsive as on other browsers

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/molecules-and-light#213]
**Test device**
Windows 7 Laptop
**Operating System**
Professional 6.1 Build
**Browser**
Firefox version 69
**Problem description**
From https://github.com/phetsims/QA/issues/419

Two finger or mouse-wheel scrolling when zoomed in is not very responsive and only moves the screen a minor amount. One other browsers as demonstrated in the visual the scrolling is more responsive, and scrolling is equally responsive on other web-pages.
**Steps to reproduce**
1. Using control and +, zoom in about 3 or 4 times so issue presents itself clearly
2. Attempt to scroll left or right using two fingers horizontally on the trackpad
3. Attempt to scroll up and down using two fingers vertically on the trackpad
4. Attempt to scroll up and down using the mouse wheel
**Visuals**
The following Visuals indicate first an attempt at trackpad scrolling on firefox compared to chrome, then a mouse-wheel scroll on firefox and then chrome.
![2019-9-12FreFxScrllingMlclght](https://user-images.githubusercontent.com/54685448/64814055-f4f7ca00-d55f-11e9-8193-6db1fd28a4cb.gif)

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-customZoom.7/phet/molecules-and-light_en_phet.html?a11y
Version: 1.5.0-customZoom.7 2019-08-28 23:16:46 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:69.0) Gecko/20100101 Firefox/69.0
Language: en-US
Window: 1366x654
Pixel Ratio: 1/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
For the same reason, I am seeing that scrolling with the mouse is just as slow.

**Comment by jessegreenberg** [comment]
Firefox has WheelEvent.deltaMode of DOM_DELTA_LINE while others default to while other browsers default to DOM_DELTA_PIXEL. Commit above determines if in "line" mode and modifies the deltas accordingly. It feels much snappier now in Firefox.

**Comment by KatieWoe** [comment]
Looks good

**Comment by jessegreenberg** [comment]
Great, thank you @KatieWoe.

**Comment by KatieWoe** [comment]
For https://github.com/phetsims/QA/issues/503 I am again noticing slow zoom and scroll in Firefox. Reopening for a second look.

**Comment by jessegreenberg** [comment]
So it seems that there is some impact of native browser zoom on scroll amount. For instance, I see in this in Firefox at 
![image](https://user-images.githubusercontent.com/6396244/84183072-5601ef00-aa59-11ea-8ece-70f7ec4e44db.png)

But the problem goes away at 100%, and gets even faster at 300%.

**Comment by jessegreenberg** [comment]
There is this firefox specific code in related to wheel
```js
    // FireFox defaults to scrolling in units of "lines" rather than pixels, resulting in slow movement - speed up
    // translation in this case
    if ( domEvent.deltaMode === window.WheelEvent.DOM_DELTA_LINE ) {
      translationX = translationX * 25;
      translationY = translationY * 25;
    }
```

**Comment by jessegreenberg** [comment]
Very strange, the speed of translation is also impacted during arrow presses.

**Comment by jessegreenberg** [comment]
OK, this has been fixed. Any panning is scaled to correct for native browser zoom or window size so that it behaves the same regardless of those factors. This was not an issue just for Firefox but for all browsers that had some zoom set to anything other than 100%. Closing.
